client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -80,7 +81,19 @@ export const setAuthCookie = (cookie: AuthCookie) => {
   Cookies.set('auth', JSON.stringify(authCookie), { expires });
 };
 
-const AuthContext = createContext<AuthControl>({} as any);
+const userHasPermission = (_permission: UserPermission) => false;
```

Comment:
Is this meant to be in this PR ?

Reply:
I had to add the dummy `authControl` for the f/e tests to run.  
I've changed to an inline constructor to make the dummy more consistent.